scripts/lmdk/tools/cmake_build.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: BSD-3-Clause
+
+
+import pathlib
+import dataclasses
+from .utils import rmtree_if_exists, execute_command
+
+
+SOF_TOP = pathlib.Path(__file__).parents[3].resolve()
+
+
+@dataclasses.dataclass
+# pylint:disable=too-many-instance-attributes
+class PlatformConfig:
+    "Product parameters"
+    vendor: str
+    PLAT_CONFIG: str
+    XTENSA_TOOLS_VERSION: str
+    XTENSA_CORE: str
+    DEFAULT_TOOLCHAIN_VARIANT: str = "xt-clang"
+    RIMAGE_KEY: pathlib.Path = pathlib.Path(SOF_TOP, "keys", "otc_private_key_3k.pem")
+    aliases: list = dataclasses.field(default_factory=list)
+    ipc4: bool = False
```

Comment:
You're copying this entire class definition but using only one constant from it?